src/tile_source.js

```diff
@@ -51,42 +38,42 @@ export default class TileSource {
     }
 
     // Mercator projection
-    static projectTile (tile) {
+    static projectData (source) {
         var timer = +new Date();
-        for (var t in tile.layers) {
-            var num_features = tile.layers[t].features.length;
+        for (var t in source.layers) {
+            var num_features = source.layers[t].features.length;
             for (var f=0; f < num_features; f++) {
-                var feature = tile.layers[t].features[f];
+                var feature = source.layers[t].features[f];
                 feature.geometry.coordinates = Geo.transformGeometry(feature.geometry, Geo.latLngToMeters);
             }
         }
 
-        if (tile.debug !== undefined) {
-            tile.debug.projection = +new Date() - timer;
+        if (source.debug !== undefined) {
+            source.debug.projection = +new Date() - timer;
         }
-        return tile;
     }
 
-    // Re-scale geometries within each tile to the range [0, scale]
-    // TODO: clip vertices at edges? right now vertices can have
-    // values outside [0, scale] (over or under bounds); this would
-    // pose a problem if we wanted to binary encode the vertices in
-    // fewer bits (e.g. 12 bits each for scale of 4096)
-    static scaleTile (tile) {
-        for (var t in tile.layers) {
-            var num_features = tile.layers[t].features.length;
+    /**
+     Re-scale geometries within each tile to the range [0, scale]
```

Comment:
This comment is kind of out of date, since `scale` is no longer passed (`Geo.units_per_meter` reflects a value computed based on scale). I should update it.